.github/dep-auditors.json

```diff
@@ -0,0 +1,4 @@
+[
+  "davidiw",
+  "mimoo"
```

Comment:
Are you not a reviewer? :)

Reply:
sounds like @bmwill is volunteering as well